### DISCUSSION TITLE
How do I style things in github readme?

### DISCUSSION THREAD

**Discussion by Kn1ghtarch** [community/community#67587] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How do I style things in github readme? For example an img.

**Comment by karaca-i** [comment]
For images, you may:
`<img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/python/python-original.svg" alt="python" width="40" height="40"/> `

You also may add some class just like in html and give it a style using the same codes for css.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
